notes/28_06_2024_tb_legacy_boot.md

```diff
@@ -0,0 +1,135 @@
+# Intro
+
+In the pursue of running new TB (Linux boot path) code on AMD, it may be useful
+to have a booting reference that used to work some years ago.
+
+Resources used:
+* Blog posts
+  - https://blog.3mdeb.com/2020/2020-08-13-trenchboot-event-log/
+  - https://blog.3mdeb.com/2020/2020-07-03-trenchboot-grub-cbfs/
+* old [meta-trenchboot repository](https://gitlab.com/trenchboot1/3mdeb/meta-trenchboot)
+* old [testing-trenchboot repository](https://gitlab.com/trenchboot1/3mdeb/testing-trenchboot)
+
+## Locating set of TB components
+
+1. [This job](https://gitlab.com/trenchboot1/3mdeb/meta-trenchboot/-/jobs/660143007)
+   claims to pass TB boot flow on a few platforms (including PC Engines APU2)
+2. This is the meta-trenchboot
+  [revision](https://gitlab.com/trenchboot1/3mdeb/meta-trenchboot/-/tree/49183efa4818987c7067ea4b35c3c25a9309b486)
+3. It translate to following components:
+   - Landing Zone (LZ) - today known as SKL
+     - [recipe](https://gitlab.com/trenchboot1/3mdeb/meta-trenchboot/-/blob/49183efa4818987c7067ea4b35c3c25a9309b486/dynamic-layers/openembedded-layer/recipes-support/landing-zone/landing-zone_0.3.0.bb)
+     - [code revision](https://github.com/TrenchBoot/landing-zone/commit/4dc904bf70d6913a5706bdfc6f7781a9ff039d38)
+  - GRUB
+    - [recipe](https://gitlab.com/trenchboot1/3mdeb/meta-trenchboot/-/blob/49183efa4818987c7067ea4b35c3c25a9309b486/dynamic-layers/openembedded-layer/recipes-bsp/grub/grub-tb_git.bb)
+    - [code revision](https://github.com/3mdeb/grub2/tree/d6fa0d1f92446243e34e009361470f5730869ee9)
+  - Linux
+    - [recipe](https://gitlab.com/trenchboot1/3mdeb/meta-trenchboot/-/blob/49183efa4818987c7067ea4b35c3c25a9309b486/dynamic-layers/openembedded-layer/recipes-kernel/linux/linux-tb_5.5.bb)
+    - [code revision](https://github.com/TrenchBoot/linux/tree/eed5cdf480ee3761d18294d64ac7e2184229b51c)
```

Comment:
This lacks indentation to align with `Landing Zone` bullet.

Reply:
7febf807068f1fbb77dc14462f9696810998f16e